pulpcore/pulpcore/app/viewsets/content.py

```diff
@@ -61,6 +61,23 @@ def create(self, request):
         headers = self.get_success_headers(serializer.data)
         return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
 
+    def destroy(self, request, pk):
+        """
+        Remove Content unit only if it is not associated with a repository version
+        """
+        repository_content_set = RepositoryContent.objects.filter(content__pk=pk)
+        
+        if repository_content_set:
+            msg = _('The Content unit cannot be deleted because it is associated with a repository '
+                    'version, which is immutable. Repository: '
+                    + repository_content_set[0].repository.name)
```

Comment:
I'm not super familiar with gettext but I think the variable needs to be outside the `_(...)`

Reply:
+1 for suggested change

Reply:
yes, use ```Repository: {repo}".format(repo=repository_content_set[0].repository.name)```